compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/packaging/converters/FilterSearch.java

```diff
@@ -47,7 +47,7 @@ public FileVisitResult visitFile(Object file, BasicFileAttributes attrs) throws
      */
     private boolean isExcluded(Path path) throws IOException {
```

Comment:
Do we need to throw IOException from here?

Reply:
This is not needed, removed it.